community-plugins.json

```diff
@@ -4185,5 +4185,12 @@
         "description": "Convert Markdown files to BBCode",
         "repo": "salockhart/obsidian-bbcode",
         "branch": "master"
+    },
+    {
+        "id": "obsidian-meta-title-filename-plugin",
+        "name": "Meta title filenames",
```

Comment:
The code looks good, but I think you might want to use the term "frontmatter" instead of "meta block." Maybe instead of "Meta title filenames," a more accurate plugin name would be "Frontmatter titles"